Tests/ComposableArchitectureTests/TestHelpers.swift

```diff
@@ -0,0 +1,10 @@
+import XCTest
+
+@available(
+  *,
+  deprecated,
+  message: "This is a test that currently fails but should not in the future."
+)
+func XCTodo(_ message: String) {
```

Comment:
Miiinot nit, but what do you think of separating the "XCT" from "Todo" and capitalizing "TODO" for case-sensitive searches:

```suggestion
func XCTTODO(_ message: String) {
```